src/cfml/system/services/JSONService.cfc

```diff
@@ -204,4 +207,47 @@ component accessors="true" singleton {
 		return props;
 	}
 
```

Comment:
Please add a java-doc style comment here and below on `isSortedJSON()` that covers the params and what they do.